q2_diversity/_alpha/_visualizer.py

```diff
@@ -146,28 +151,28 @@ def alpha_correlation(output_dir: str,
     g.savefig(os.path.join(output_dir, 'scatter-plot.png'))
     g.savefig(os.path.join(output_dir, 'scatter-plot.pdf'))
 
-    index_fp = os.path.join(output_dir, 'index.html')
-    with open(index_fp, 'w') as fh:
-        fh.write('<html><body>')
-        if alpha_diversity.shape[0] != df.shape[0]:
-            fh.write("<b>Warning</b>: Some samples were filtered because they "
-                     "were missing metadata values.<br><b>The input "
-                     "contained %d samples but %s was computed on "
-                     "only %d samples.</b><p>"
-                     % (alpha_diversity.shape[0], method, df.shape[0]))
-        fh.write('<table border=1>\n')
-        fh.write(' <tr><td>Test</td><td>%s correlation</td></tr>\n' %
-                 method.title())
-        fh.write(' <tr><td>Test statistic</td><td>%1.4f</td></tr>\n' %
-                 correlation_result[0])
-        fh.write(' <tr><td>P-value</td><td>%1.4f</td></tr>\n' %
-                 correlation_result[1])
-        fh.write(' <tr><td>Sample size</td><td>%d</td></tr>\n' %
-                 df.shape[0])
-        fh.write('</table>')
-        fh.write('<p>\n')
-        fh.write('<a href="scatter-plot.pdf">\n')
-        fh.write(' <img src="scatter-plot.png">')
-        fh.write(' <p>Download as PDF</p>\n')
-        fh.write('</a>\n\n')
-        fh.write('</body></html>')
+    warning = None
```

Comment:
You've tested with and without a warning, right? Just want to confirm because I notice that the data type differs when there are/aren't warnings. The unit tests would catch this as it covers both cases.